storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2656,11 +2656,12 @@ static bool xtrabackup_read_info(char *filename) {
 
   char lock[8];
   if (fscanf(fp, "lock_ddl_type = %7s\n", lock) != 1) {
-    r = false;
-    goto end;
+    xb::warn() << "lock_ddl_type parameter not found, defaulting to 'ON'";
+    opt_lock_ddl = LOCK_DDL_ON;
+  } else {
+    /* used at log0recv.cc to apply or not file operations */
```

Comment:
You mean, this option is used to handle the metadata files created by backup (lock-ddl=reduced)?